Gemfile

```diff
@@ -6,6 +6,7 @@ gem 'fastlane', '~> 2'
 gem 'rake', '~> 12.3'
 gem 'xcode-install'
 gem 'xcpretty-travis-formatter'
+gem 'cocoapods-catalyst-support'
```

Comment:
Can we put a version constraint on this? I see that `0.2.0` shipped a few days back, so pinning this for now might be good to ensure no accidental breakage.

Reply:
+1 👍 .
Even if the lockfile (`Gemfile.lock`) is what pins this to a specific version accross `gem install`s (and will only risk bumping if we `gem update`), I agree that we should at least semver-constrain it to `~> 0.1` to constrain it during other gems' updates.